parachains/runtimes/collectives/collectives-polkadot/src/lib.rs

```diff
@@ -519,10 +563,18 @@ construct_runtime!(
 		Utility: pallet_utility::{Pallet, Call, Event} = 40,
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 41,
 		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 42,
+		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 43,
+		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 44,
```

Comment:
We don't really need to enumerate all the parts of the pallet, let's just do:
```suggestion
		Preimage: pallet_preimage = 43,
		Scheduler: pallet_scheduler = 44,
```

Reply:
I know it's controversial, but omitting them pulls in everything, including stuff you don't need (Genesis, Origins). I prefer to list them explicitly.